### PR TITLE
Issue 20 and 3 - bump versions minio postgres and added documentation

### LINKED ISSUE #3: Add readme

### LINKED ISSUE #20: Bump modules versions
- [x] minio
- [x] postgres

### PR DESCRIPTION
**Pull request by claesgill**
Closes #20 
Closes #3 

- Bumped versions minio and postgres
- Added documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,155 @@
...
+  ]
+}
+```
+
+### Verifying setup
+You can verify the setup by connection to minio with a consul proxy. Check out the [required software](#required-software) section.
+```sh
+make proxy-to-minio
+```
+Minio should be available at [localhost:9000](http://localhost:9000/minio/).
```
Comment (zhenik) [inline README.md:152]: I think this part should be different. How to verify hive setup

**Review by zhenik — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (claesgill) [reply README.md:152]: Yes, you are probably right. Do you have any suggestions?

Comment (zhenik) [reply README.md:152]: You need query tables via beeline and consul proxy. Hint here https://github.com/zhenik-poc/big-data-stack-practice#useful-commands-debug

Comment (claesgill) [reply README.md:152]: Alright, thanks!

File: README.md [hunk 2]
```diff
@@ -145,11 +145,19 @@ module "hive" {
 ```
 
 ### Verifying setup
-You can verify the setup by connection to minio with a consul proxy. Check out the [required software](#required-software) section.
+You can verify the setup by connection to Hive using the Nomad UI at [localhost:4646](http://localhost:4646/). Follow the steps below.
+1. Locate and click the *hive-metastore* service.
+2. Click the *exec* button and connect to the *metastoreserver* task.
+3. Run `beeline -u jdbc:hive2://` to connect to hive.
+4. Run `SHOW databases;`. Your output should look something like this:
```
Comment (zhenik) [inline README.md:152]:
```suggestion
4. Run `SHOW databases;`. Your output should look like this:
```

**Review by zhenik — APPROVED** [review]

**Review by dangernil — APPROVED** [review]

**Pull request merged**